main/src/main/java/cgeo/geocaching/ui/SimpleItemListModel.java

```diff
@@ -475,6 +476,23 @@ public Set<T> getSelectedItems() {
         return selectedItems;
     }
 
+    /** Sets the currently selected items */
+    public SimpleItemListModel<T> setDisabledItems(final Iterable<T> disabled) {
+        if (disabled != null) {
+            disabledItems.clear();
+            for (T sel : disabled) {
+                disabledItems.add(sel);
+            }
+        }
```

Comment:
I am almost sure you need to fire a Change Event here, so list is re-rendered if disabled items are changed after  dialog is already shown (eg programatically after button tap or something like this). "triggerChange(ChangeType.COMPLETE)" should do it (will re-render the whole table, but I assume disabled is not used often enough for special handling).

You can try this out eg by calling "setDisabledItems" AFTER "selectSingle/Multiple" was called

Reply:
Also gives the correct result